### PR TITLE
Use custom test options for a smaller Elastic footprint during testing.

### PR DESCRIPTION
**Pull request by benclarkwood**
Elastic's defaults are too large to fit on a m4.large. Set smaller defaults during tests.

### CODE REVIEW

File: frameworks/elastic/tests/config.py [hunk 1]
```diff
@@ -47,6 +47,38 @@
                 "role": {"type": "keyword"}}}}}
 
 
+def install(
+        package_name,
+        service_name,
+        expected_running_tasks,
+        additional_options={},
+        package_version=None,
+        timeout_seconds=5*60,
```
Comment (loren) [inline frameworks/elastic/tests/config.py]: That sure seems optimistic. I suggest 25 minutes as we do in `test_upgrade`.

File: frameworks/elastic/tests/config.py [hunk 2]
```diff
@@ -47,6 +47,38 @@
...
+        wait_for_deployment=True):
+    test_options={
+        "master_nodes": {
+            "cpus": 0.25
+        },
+        "date_nodes": {
```
Comment (loren) [inline frameworks/elastic/tests/config.py]: data

File: frameworks/elastic/tests/test_sanity.py [hunk 3]
```diff
@@ -30,8 +30,21 @@ def configure_package(configure_security):
             foldered_name,
             current_expected_task_count,
             additional_options={
-                "service": {"name": foldered_name},
-                "ingest_nodes": {"count": 1} })
+                "master_nodes": {
+                    "cpus": 0.25
+                },
+                "date_nodes": {
```
Comment (loren) [inline frameworks/elastic/tests/test_sanity.py]: data

**Comment by benclarkwood** [comment]
@loren fixed.

**Comment by benclarkwood** [comment]
Closing temporarily so TC wont trigger.

**Pull request closed without merging**